playbooks/ceph.yml

```diff
@@ -83,10 +83,35 @@
   hosts: "{{ cifmw_ceph_target | default('computes')  }}"
   gather_facts: true
   become: true
-  vars:
-    ansible_ssh_private_key_file: "{{ lookup('env', 'ANSIBLE_SSH_PRIVATE_KEY') }}"
-  roles:
-    - role: cifmw_block_device
+  tasks:
+    - name: set cifmw_num_osds_perhost
```

Comment:
In order to pass pre-commit tests, `All names should start with an uppercase letter`
This fails multiple times in the following files:
```
playbooks/ceph.yml:87: name[casing]: All names should start with an uppercase letter.
playbooks/ceph.yml:99: name[casing]: All names should start with an uppercase letter.
roles/cifmw_ceph_spec/tasks/main.yml:17: name[casing]: All names should start with an uppercase letter.
roles/cifmw_ceph_spec/tasks/main.yml:30: name[casing]: All names should start with an uppercase letter
```

Care to correct? It should help passing Prow jobs.

Reply:
Sure, i will fix it in the next patch.